Library/Edge.cs

```diff
@@ -35,28 +39,34 @@ public class Edge
         public Edge prev2;
         public Loop loop; // first node of the list of faces that use this edge. Navigate list using radial_next
 
-        /**
-         * Tells whether a vertex is one of the extremities of this edge.
-         */
+        /// <summary>
+        /// Whether the specified <see cref="Vertex"/> is one of the vertices that make up the <see cref="Edge"/>.
+        /// </summary>
+        /// <param name="v">The <see cref="Vertex"/> to compare.</param>
+        /// <returns><c>true</c> if <paramref name="v"/> is used by the <see cref="Edge"/>; otherwise, <c>false</c>.</returns>
         public bool ContainsVertex(Vertex v)
         {
             return v == vert1 || v == vert2;
         }
 
-        /**
-         * If one gives a vertex of the edge to this function, it returns the
-         * other vertex of the edge. Otherwise, the behavior is undefined.
-         */
+        /// <summary>
+        /// Returns the other <see cref="Vertex"/> that makes up the <see cref="Edge"/> of the specified <see cref="Vertex"/>.
+        /// </summary>
+        /// <remarks>Assumes that the specified <see cref="Vertex"/> is one of the vertices that make up the <see cref="Edge"/>; otherwise, behavior is undefined.</remarks>
```

Comment:
> `... that make up the <see cref="Edge"/>, i.e. that <see cref="ContainsVertex"/> returns <c>true</c>. Otherwise, **the** behavior is undefined.`

That seems implicitly stated, more so if you read the `ContainsVertex` comment first. Also I think it is best to avoid referencing the behavior of other methods because they could change or be removed and the comment would then be inaccurate.